acceptance-tests/src/main/kotlin/org/ostelco/at/jersey/Tests.kt

```diff
@@ -256,6 +257,94 @@ class SourceTest {
         val cardId = StripePayment.getCardIdForTokenId(sourceId)
         assertNotNull(sources.first { it.id == cardId }, "Expected card $cardId in list of payment sources for profile $email")
     }
+
+    @Test
+    fun `okhttp test - GET list sources`() {
+
+        StripePayment.deleteAllCustomers()
+        Firebase.deleteAllPaymentCustomers()
+
+        val email = "purchase-${randomInt()}@test.com"
+        createProfile(name = "Test Payment Source", email = email)
+
+        val sourceId = StripePayment.createPaymentTokenId()
```

Comment:
rename to tokenId

Reply:
Good point! But this is also inconsistent in the API itself I think. Should perhaps be cleaned up?